index.rst

```diff
@@ -1,57 +1,343 @@
+In the majority of cases, these routine operations are expected to be executed by the operator from a high-level interface of the observatory control software (as discussed in following sections).
+
+During integration, commissioning, and engineering time, there will be situations where non-standard, customized operations will need to be executed.
+These operations will often be exercised by specialists to a specific subsystem (e.g. M1M3) and require a knowledge base that differs from that of an observing specialist.
+The breadth of the observatory control system allows users to perform both standard routines and customized tasks via three main interactive components:
+
+    - :ref:`LSST Observing Visualization Environment (LOVE)<LOVE>`
+    - :ref:`ScriptQueue <Script-Queue>`
+    - :ref:`Jupyter notebooks <Jupyter-notebooks>`
+
+This document begins with a high-level overview of the three components and explores how users can take advantage of their features to accomplish different types of tasks.
+Then, we define three main actors and outline their roles and assumed levels of knowledge regarding when performing specific tasks.
+These actors and roles are used to assist in defining the usability expectations of the components and control system in general as well as the underlying assumptions of the knowledge required for each role.
+In reality, the people's roles will be less discrete and there will be significant overlap between these roles.
+The identified actors are:
+
+    1. Observing Specialists
+    2. Commissioning Scientists
+    3. Software Developers
+
+Finally, we present concrete examples of interactions that the actors will perform with the OCS, linking to procedures on how to do them using the multiple components.
+These examples explore how users best take advantage of the component's features.
+They also include cases of how *not* to use a component due limitations or inefficiencies in various areas.
+The examples are also useful in facilitating discussion regarding the ease-of-use of each operation, which is expected to stimulate discussion on how to best improve system usability and/or performance.
+
+.. _LOVE:
+
+LOVE
+====
+
+Tiago insert (or link) content here.
+
+Important to say that users will interact with basic observatory operations via LOVE, but what is actually happening on the back end is that these operations (e.g. offsets) are actually scripts that get loaded and performed.
+
+
+.. _Script-Queue:
+
+Script Queue
+============
+
+The `ScriptQueue`_ is the main hub for the execution of regular operations and requires a minimum level of interaction by the user.
+As mentioned in the previous section, it is also used as a back-end for user-commanded actions performed using the `LOVE`_ interface.
+For example, when a user performs a telescope offset operation using LOVE, the actual action perform by LOVE system will the queueing of a :ref:`SAL Script <SAL_Scripts>` that then gets executed by the `ScriptQueue`_.
```

Comment:
I think we need to clarify something here. LOVE can also send commands directly to CSCs that does not pass through the ScriptQueue. For instance, LOVE can acknowledge alarms from the Watcher directly. Also, the current "TCS" views will use the `TCS` class directly to execute tasks like offsets, stop and emergency stop.

Reply:
I knew that was true for the E-stop but not the others.
So what is a good example of a user-interaction that does load a script as opposed to using the TCS. Taking an image?

Reply:
I changed the example to taking an image, and added a note on line 66 in the LOVE section to discuss the difference between when LOVE interacts with the TCS and when it interacts with the scriptQueue.